### PR TITLE
[WEB-8448] Added file upload endpoints to sws-py-sdk

### PR DESCRIPTION
**Pull request by clea51**
[WEB-8448] Added file upload endpoints to sws-py-sdk

[WEB-8448]: https://serato.atlassian.net/browse/WEB-8448?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: tests/integration/cloudlib_test.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# from sws_py_sdk import cloudlib, sws_client
+from base_test import me_endpoint_sws_client, user_endpoint_sws_client
+
+def me_create_file_upload(me_endpoint_sws_client):
+    response = me_endpoint_sws_client.cloudlib().me_create_file_upload(md5_hash='mockhash', mime_type='mockmime', size=123, name=None)
+    assert response.status_code != 404
+    assert response.status_code != 500
+
+def test_user_create_file_upload(user_endpoint_sws_client):
+    response = user_endpoint_sws_client.cloudlib().user_create_file_upload(md5_hash='mockhash', mime_type='mockmime', size=123, name=None)
```
Comment (ZhuchenLiu) [inline tests/integration/cloudlib_test.py:19]: Missing `user_id` here

File: tests/integration/base_test.py [hunk 2]
```diff
@@ -6,7 +6,8 @@
 SERVICE_URI= {
     "id": "http://192.168.4.14:8585",
     "license": "http://192.168.4.14:8686",
-    "ecom": "http://192.168.4.14:8787"
+    "ecom": "http://192.168.4.14:8787",
+    "cloudlib": "localhost:3000"
```
Comment (ZhuchenLiu) [inline tests/integration/base_test.py:10]: We need to add `http://` before the localhost

**Review by ZhuchenLiu — COMMENTED** [review]
Well done. @clea51 Just two slight issues need to be fixed.

File: tests/integration/base_test.py (hunk 2 above)
Comment (clea51) [reply tests/integration/base_test.py]: Thanks heaps @ZhuchenLiu I have fixed these and updated the branch

**Review by ZhuchenLiu — APPROVED** [review]
All tests passed in my machine.
image

**Review by takayuki-serato — APPROVED** [review]
Nice one @clea51 ! Could you just update the PR title with what we did here in the format `[WEB-8448] My Description Here`?

**Comment by clea51** [comment]
Thanks heaps @takayuki-serato ! Can you check the description for me please? I've edited the original comment and hoping that's the correct place for a description but not sure (let me know if there's another place to do it plz)

**Comment by takayuki-serato** [comment]
Looks good @clea51 , go ahead and merge it to master. Leave the deployment to PyPi to me.

**Pull request merged**